README.md

```diff
@@ -139,6 +139,115 @@ $ near
 
 The CLI interactively guides you through some pretty complex topics, helping you make informed decisions along the way.
 
+## Shell Configuration for Command History
```

Comment:
this doc section can be linked to in a separate md file. 
this usually improves readability, as there's less of a shock on a reader, 
as he/she doesn't undergo reading all of the details at once, but sees document structure first, 
and can narrow his/her context and attention on a specific section later

Reply:
Yep, makes sense. I was thinking about that, but decided just to put everything at the bottom for some reason. Fixed in 9b5687087718b885b0b986c80e80089a54a77652